lizmap/www/assets/css/admin.css

```
@@ -120,6 +120,12 @@ table.lizmap_project_list tr {
 table.lizmap_project_list tr.selected {
     background-color: lightskyblue;
 }
+table.lizmap_project_list tr.active a,
+table.lizmap_project_list tr.active a:hover,
+table.lizmap_project_list tr.selected a,
+table.lizmap_project_list tr.selected a:hover{
+  color: #005580;
```

I would have kept the white + underline the text
```suggestion
  color: white;
  text-decoration: underline dotted;
```